src/actions/types.js

```diff
@@ -20,3 +20,8 @@ export const SITE_NEAREST_TOWN_TEXT_CHANGED = 'site_nearest_town_text_changed';
 export const SITE_ACCESSIBILITY_OPTION_CHANGED = 'site_accessibility_option_changed';
 export const SITE_FACILITIES_OPTION_CHANGED = 'site_facilities_option_changed';
 export const ADD_SITE_FIELDS_RESET = 'add_site_fields_reset';
+export const CURRENT_LOCATION_UPDATED = 'current_location_updated';
+
+export const LOCATION_SERVICES_PERMISSION_UPDATED = 'location_services_permission_updated';
+export const CAMERA_PERMISSION_UPDATED = 'camera_permission_updated';
+export const CAMERA_ROLL_PERMISSION_UPDATED = 'camera_roll_permission_updated';
```

Comment:
Added some additional permissions types in advance of the work we'll be doing anyway.